docs/getting_started/debugging_t8n_tools.md

```diff
@@ -0,0 +1,101 @@
+# Debugging Transition Tools
+
+The `--t8n-dump-dir` flag can be used to dump the inputs and outputs of every call made to the `t8n` command to help debugging or simply understand how a test is interacting with the EVM.
+
+In particular, a script `t8n.sh` is generated for each call to the `t8n` command which can be used to reproduce the call to trigger errors or attach a debugger without the need to execute Python.
+
+For example, running:
+
+```console
+fill tests/berlin/eip2930_access_list/ --fork Berlin \
+    --t8n-dump-dir=/tmp/evm-t8n-dump
+```
+
+will produce the directory structure:
+
+```console
+/tmp/evm-t8n-dump/
+└── test_access_list_fork_Berlin
+    ├── 0
+    │   ├── args.py
+    │   ├── input
+    │   │   ├── alloc.json
+    │   │   ├── env.json
+    │   │   └── txs.json
+    │   ├── output
+    │   │   ├── alloc.json
+    │   │   ├── result.json
+    │   │   └── txs.rlp
+    │   ├── returncode.txt
+    │   ├── stderr.txt
+    │   ├── stdin.txt
+    │   ├── stdout.txt
+    │   └── t8n.sh
+    └── 1
+        ├── args.py
+        ├── input
+        │   ├── alloc.json
+        │   ├── env.json
+        │   └── txs.json
+        ├── output
+        │   ├── alloc.json
+        │   ├── result.json
+        │   └── txs.rlp
+        ├── returncode.txt
+        ├── stderr.txt
+        ├── stdin.txt
+        ├── stdout.txt
+        └── t8n.sh
+```
+
+where the directories `0` and `1` correspond to the different calls made to the `t8n` tool executed during the test, in this case (pre-Shanghai state):
+
+- `0` corresponds to the call used to calculate the state root of the test's initial alloc (which is why it has an empty transaction list).
+- `1` corresponds to the call used to execute the first transaction from the test.
+
+For post-Shanghai forks:
+
+- `0` corresponds to the call used to calculate the `stateRoot` for the test's initial alloc.
+- `1` (optional) corresponds to the call used to calculate the initial `withdrawalsRoot` for the test's initial alloc if required.
```

Comment:
Let's just merge https://github.com/ethereum/execution-spec-tests/pull/273 first and get rid of this once and for all :)